README.md

```diff
@@ -372,6 +376,7 @@ test:
 
     # Step for Minitest
     - KNAPSACK_GENERATE_REPORT=true bundle exec rake test
+    - KNAPSACK_GENERATE_REPORT=true bundle exec rake test # For rails 5.1 with system tests
```

Comment:
@rubendinho I've updated example in docs here:
https://github.com/KnapsackPro/docs.knapsackpro.com/commit/13b144a9a755cd7e9ac348a26cce718cf944a588
https://github.com/KnapsackPro/docs.knapsackpro.com/commit/0a4d46c7ec27e94a0915b316098495038804c061

I hope this will be more understandable now :)

Thanks for good catch!

BTW If you would like to learn more cool things how to run fast tests with dynamic tests allocation please check https://youtu.be/gJdLQb83hho